packages/react-forms/src/InputDate/__mocks__/InputDateWrapper.js

```diff
@@ -0,0 +1,86 @@
+/**
```

Comment:
Why all this code for a mock?

Reply:
I need it because I wanted to test state changes, calendar arrows, date picking from calendar, etc. Mainly to be able to test what we are fixing now. Those tests live in `InputDateWithCalendar.test.js`.

Reply:
I simplified it a lot and removed it.